src/components/atoms/markdown/CodeFence.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import styled from '../../../lib/styled'
 import copy from 'copy-to-clipboard'
 import Icon from '../Icon'
-import { mdiContentCopy } from '@mdi/js'
+import { mdiContentCopy, mdiContentSave } from '@mdi/js'
 import { flexCenter } from '../../../lib/styled/styleFunctions'
-
+import { saveAs } from 'file-saver';
```

Comment:
As I said in the previous PR, I don't think we need file-saver library!

Reply:
As I said 
if use `writeFile `and `showOpenDialog`   maybe it can't be save to file on the web or mobile, I think file saver is better.
it's html5 function js